src/ui/control/geolocate_control.js

```diff
@@ -78,6 +80,7 @@ let noTimeout = false;
  * @param {Object} [options.positionOptions={enableHighAccuracy: false, timeout: 6000}] A Geolocation API [PositionOptions](https://developer.mozilla.org/en-US/docs/Web/API/PositionOptions) object.
  * @param {Object} [options.fitBoundsOptions={maxZoom: 15}] A [`fitBounds`](#map#fitbounds) options object to use when the map is panned and zoomed to the user's location. The default is to use a `maxZoom` of 15 to limit how far the map will zoom in for very accurate locations.
  * @param {Object} [options.trackUserLocation=false] If `true` the Geolocate Control becomes a toggle button and when active the map will receive updates to the user's location as it changes.
+ * @param {Object} [options.showAccuracy=true] If trackUserLocation is true, by default a transparent circle will be drawn around the user location indicating the accuracy (95% confidence level) of the user's location. Set to `false` to disable.
```

Comment:
"If trackUserLocation is true" is that true though, trackUserLocation just means that it uses watchPosition to update the location continuously. If trackUserLocation is false it will just do a single location request each time you press the button, so even if trackUserLocation=false you may still want this accuracy circle.

Reply:
Good catch this should have been `showUserLocation`